Updates for OpenFHE (#48)
jonaschn/awesome-he: README.md

- [NFLlib](https://github.com/quarkslab/NFLlib) - NTT-based Fast Lattice library specialized on power-of-two polynomials.
- [node-seal](https://github.com/morfix-io/node-seal) - JavaScript/WebAssembly port of [Microsoft SEAL](#SEAL).
- [NuFHE](https://github.com/nucypher/nufhe) - GPU-accelerated HE library, faster than cuFHE, that implements the [tfhe](#tfhe) algorithms.

> - <a name="OpenFHE">[OpenFHE](https://github.com/openfheorg/openfhe-development) - C++ FHE library implementing all major schemes along with bootstrapping and scheme switching capabilities.

Reviewer: ```suggestion
- <a name="OpenFHE">[OpenFHE](https://github.com/openfheorg/openfhe-development) - C++ FHE library implementing all major schemes along with bootstrapping and scheme switching.
```

Reviewer: Otherwise description is too long (should be single line)

> - <a name="OpenFHE">[OpenFHE](https://github.com/openfheorg/openfhe-development) - C++ FHE library implementing BFV, BGV, CKKS, DM/FHEW, and CGGI/TFHE schemes along with bootstrapping and scheme switching capabilities.

Reviewer: ```suggestion
- <a name="OpenFHE">[OpenFHE](https://github.com/openfheorg/openfhe-development) - C++ FHE library implementing all major schemes along with bootstrapping and scheme switching capabilities.
```

- [lattigo-polls](https://github.com/ldsec/lattigo-polls-demo) - Web-application for scheduling meetings using [lattigo](#lattigo).
- [Morfix.io](https://morfix.io/sandbox) - Web-based UI to play around with the [Microsoft SEAL](#SEAL) library.
- [nGraph-HE](https://github.com/IntelAI/he-transformer) - Deep Learning (DL) with HE through Intel’s DL graph compiler nGraph based on [SEAL](#SEAL).

> - [OpenFHE demo applications](https://github.com/openfheorg/openfhe-development):

Reviewer: ```suggestion
- [OpenFHE demo applications](https://github.com/openfheorg/openfhe-development) - Several demo applications that demonstrate some of the capabilities of OpenFHE
```
